### ISSUE TITLE
Moving intralink validation checks to a log file

### ISSUE THREAD

**Issue by wxwxwxwx9** [MarkBind/markbind#1470]
**Is your request related to a problem?**

(Write your answer here.)
This idea came about after a discussion with @damithc. We brought up that user experience can potentially be impacted when there are many invalid link checks when generating the site, thus flooding the console and user may miss out on error messages. 

**Describe the solution you'd like**

I was thinking if outputting the invalid links to a log file instead of printing on the console would be a way to alleviate the problem? We will just print in the console to alert user that we have found some potentially invalid links and direct them to the log file so they can see what links may be broken. 

(Describe your proposed solution here.)

**Describe alternatives you've considered**

(Write your answer here.)

**Additional context**

(Write your answer here.)

**Comment by damithc** [comment]
Good idea to explore this direction. What is the plan for the live preview mode?

A contributing factor to this problem is false positives. If the warnings are all _real_ ones (i.e., no false positives) the user should either fix the broken links or he can switch off the link checking. You might want to try the CS2103 website and see if the warnings you see are all real ones that need to be fixed.
